Testscripts/Linux/docker_java_app.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
```

Comment:
Both programs look similar. Can you merge with different parameter?

Reply:
The intention is to test application in different language. The environment is driver by docker file. Docker file specifies the packages which needs to be installed and platform variant.